mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -95,14 +99,13 @@ public boolean hasService(String serviceName) {
 
   @SuppressWarnings("unchecked") //
   private <T> T findService(String serviceName) {
-    if (!isDestroyed()) {
-      if (SERVICE_NAME.equals(serviceName)) return (T) this;
+    assertNotDead();
```

Comment:
I disagree with this change. getSystemService is too fundamental system for us to intrude on like this. I'd particularly expect it to cause problems during Activity teardown.